ext/reflection/tests/bug74351.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+Bug #74351 (Wrong reflection on variadic array functions)
+--FILE--
+<?php
+$functions = [
+    'array_diff',
```

Comment:
Could you add a userland declared example function to the list, so we can see if the behaviour is consistent with/without patch?